recipes/cyanure/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.22.4" %}
+{% set tag = "v0.0.4" %}
+
+package:
+  name: "cyanure"
+  version: {{ version }}
+
+source:
+  url: https://github.com/thomasryck/cyanure/archive/{{ tag }}/cyanure-{{ version }}.tar.gz
+  sha256: f926e613873221bcc96438f3a50379d8ec30616e79ac5c15476edc6bf45b7b2e
+    
+build:
+  number: 0
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - python
+    - libgomp  # [linux]
+    - intel-openmp  # [win]
+  host:
+    - python
+    - libblas
+    - libcblas
+    - numpy >=1.18.0
+    - scipy
+    - scikit-learn
+    - pip
+  run:
+    - python
+    
+
+test:
+  imports:
+    - cyanure
+  source_files:
+   - test/*.py
+  requires:
```

Comment:
You should remove all runtime dependencies of the library from here and just list `pytest`.